pkg/controller/operatingsystemconfig/oscommon/actuator/actuator_util.go

```diff
@@ -67,10 +67,11 @@ func CloudConfigFromOperatingSystemConfig(ctx context.Context, cli runtimeclient
 	}
 
 	return generator.Generate(&commonosgenerator.OperatingSystemConfig{
-		Bootstrap: config.Spec.Purpose == extensionsv1alpha1.OperatingSystemConfigPurposeProvision,
-		Files:     files,
-		Units:     units,
-		Path:      config.Spec.ReloadConfigFilePath,
+		Bootstrap:           config.Spec.Purpose == extensionsv1alpha1.OperatingSystemConfigPurposeProvision,
+		Files:               files,
+		Units:               units,
+		Path:                config.Spec.ReloadConfigFilePath,
+		IsContainerDEnabled: config.Spec.CRIConfig != nil && config.Spec.CRIConfig.Name == extensionsv1alpha1.CRINameContainerD,
```

Comment:
But why is this merged when this PR depends on a PR in g/g which is still not merged?
And the ci/cd checks passed on a compilation issue?

Reply:
```
Executing golangci-lint
WARN [runner] Can't run linter unused: buildssa: analysis skipped: errors in package: [/tmp/build/86a4261d/pull-request-gardener_gardener-extensions-pr_master/pkg/controller/operatingsystemconfig/oscommon/actuator/actuator_util.go:74:36: config.Spec.CRIConfig undefined (type v1alpha1.OperatingSystemConfigSpec has no field or method CRIConfig) /tmp/build/86a4261d/pull-request-gardener_gardener-extensions-pr_master/pkg/controller/operatingsystemconfig/oscommon/actuator/actuator_util.go:74:68: config.Spec.CRIConfig undefined (type v1alpha1.OperatingSystemConfigSpec has no field or method CRIConfig) /tmp/build/86a4261d/pull-request-gardener_gardener-extensions-pr_master/pkg/controller/operatingsystemconfig/oscommon/actuator/actuator_util.go:74:105: CRINameContainerD not declared by package v1alpha1] 
WARN [runner] Can't run linter goanalysis_metalinter: assign: failed prerequisites: inspect@github.com/gardener/gardener-extensions/pkg/controller/operatingsystemconfig/oscommon/actuator 
```